plow/backends/js/js_natives.flow

```diff
@@ -161,6 +161,10 @@ jsBackend() -> Backend {
 			BackGen("Native.getCharCodeAt", bTypeVoid, BackText("function(s,i){return s.charCodeAt(i);}")),
 			BackGen("Native.getCharAt", bTypeVoid, BackText("function(s,i){return s.charAt(i);}")),
 
+			BackGen("Native.toLowerCase", bTypeString, BackText("Native.toLowerCase")),
+			BackGen("Native.toUpperCase", bTypeString, BackText("Native.toUpperCase")),
```

Comment:
We should find a way where this is  not required. Natives not found in this table should all fall back to the haxe runtime

Reply:
ok